packages/legend-extension-dsl-persistence/src/graph-manager/protocol/pure/v1/model/packageableElements/persistence/V1_DSL_Persistence_Deduplication.ts

```diff
@@ -36,8 +37,12 @@ export class V1_AnyVersion extends V1_Deduplication {
 export abstract class V1_MaxVersion extends V1_Deduplication {}
 
 export class V1_MaxVersionForGraphFetch extends V1_MaxVersion {
+  versionFieldPath!: object;
```

Comment:
same comment, please annotate this discrepancy

Reply:
Updated.